lib/abci/BlockExecutionDBTransactions.js

```diff
@@ -0,0 +1,43 @@
+class BlockExecutionDBTransactions {
+  constructor() {
+    this.transactions = {
+      identity: null,
+    };
+  }
+
+  /**
+   * Set identity DB transaction
+   *
+   * @param {LevelDBTransaction} transaction
+   * @return {BlockExecutionDBTransactions}
+   */
+  setIdentityTransaction(transaction) {
+    this.transactions.identity = transaction;
+
+    return this;
+  }
+
+  /**
+   * Get identity DB transaction
+   *
+   * @return {LevelDBTransaction}
+   */
+  getIdentityTransaction() {
+    return this.transactions.identity;
+  }
+
+  /**
+   * Commit all transactions
+   *
+   * @return {Promise<Transaction>}
```

Comment:
`Promise<[LevelDBTransaction]>` ?

Reply:
No, because it will be result of commit

Reply:
Are you sure? https://github.com/dashevo/js-machine/pull/13/files#diff-5c076bdb6a3dd9d184d90c947493426dR52

Reply:
` const db = level('./db/transaction-test', { valueEncoding: 'binary' });
    const transaction = new LevelDBTransaction(db);
    transaction.startTransaction();
    const blockExecutionDBTransactions = new BlockExecutionDBTransactions();
    blockExecutionDBTransactions.setIdentityTransaction(transaction);
    const result = await blockExecutionDBTransactions.commit();
    console.log(result);`

`[undefined]`